test/functional/cmdLineTests/verbosetest/playlist.xml

```diff
@@ -49,4 +49,34 @@
 			<impl>ibm</impl>
 		</impls>
 	</test>
+
+	<test>
+		<testCaseName>cmdLineTester_verbosetest_11</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+			<variation>Mode501</variation>
```

Comment:
We don't need to test all these modes. The code doesn't depend on JIT modes, GC policy, compressed or non-compressed. Testing NoOptions should be sufficient.